reflection.md

```diff
@@ -6,4 +6,4 @@ on the following question:
 Is there anything that is still annoying or difficult about the operations we
 have in the library (i.e., flipping, rotating, and grayscaling)?
 
-_replace this text with your answer_
+I still felt like the code itself is strenuous to understand, and I still felt like a lot of the program can be simplified in the sense that a lot of the code are "repeated".
```

Comment:
Yes, a lot of the `Picture` library implementation is repeated, and could be simplified. This question was also intended to get at the _user's_ perspective: From the user's point of view (e.g,. when writing `Practice.scala`, is there anything still annoying about the library?

Reply:
Additionally, this reflection could be improved with a bit more specificity: What in particular is strenuous to understand, and what parts can be simplified?